src/test/java/com/bearcat/eventplanner/AppTest.java

```diff
@@ -0,0 +1,32 @@
+package com.bearcat.eventplanner;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+
+@SpringBootApplication
+class AppTests {
+
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final PrintStream originalOut = System.out;
+
+    @BeforeEach
+    void setUpStreams() {
+        System.setOut(new PrintStream(outContent));
+    }
+
+    @AfterEach
+    void restoreStreams() {
+        System.setOut(originalOut);
+    }
+
+    @Test
+    void contextLoads() {
+
+    }
+}
```

Comment:
Add the testing package that ensures the Context loads of the application run properly of the Spring Application run correctly and generate no errors. The test completes as intended and passes. This decreases the technical debt and avoids the 7 deadly sins of lack of testing, https://www.techopedia.com/definition/27913/technical-debt, https://www.methodsandtools.com/tools/tools.php?sonar#:~:text=In%20the%20Sonar%20team%2C%20we%20like%20to%20call,bugs%207%20having%20a%20spaghetti%20design%20%28package%20cycles...%29